package.json

```diff
@@ -12,6 +12,9 @@
         "branch": "",
         "SHA": ""
     },
+    "dependencies": {
+        "chokidar": "1.6.0"
+    },
```

Comment:
We should start using a npm srinkwrap file asap if we start to use the dependecies fields.